python/pyspark/sql/dataframe.py

```diff
@@ -81,6 +82,7 @@ def __init__(self, jdf, sql_ctx):
         # Check whether _repr_html is supported or not, we use it to avoid calling _jdf twice
         # by __repr__ and _repr_html_ while eager evaluation opened.
         self._support_repr_html = False
+        self.sql_conf = SQLConf(sql_ctx)
```

Comment:
BTW, does Scala side has such attribute? If no, I won't do it in this way.

Reply:
Got it, I'll try to do this by wrapping existing SQLConf.